<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="4" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-19.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-19.png"height="100"width="100"/></a>]]>
    </line>
    <line>
      <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-57.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-57.png"height="100"width="100"/></a>]]>
    </line>
    <line>
      <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-09-00-44.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-09-00-44.png"height="100"width="100"/></a>]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="122409" started-at="2021-03-21T20:58:46 IST" finished-at="2021-03-21T21:00:48 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="122409" started-at="2021-03-21T20:58:46 IST" finished-at="2021-03-21T21:00:48 IST">
      <class name="Tests.MailSentTestwithDp">
        <test-method status="PASS" signature="setup()[pri:0, instance:Tests.MailSentTestwithDp@e874448]" name="setup" is-config="true" duration-ms="209" started-at="2021-03-21T20:58:46 IST" finished-at="2021-03-21T20:58:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="mailSentTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:Tests.MailSentTestwithDp@e874448]" name="mailSentTest" duration-ms="34479" started-at="2021-03-21T20:58:46 IST" data-provider="providerMethod" finished-at="2021-03-21T20:59:20 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[magnitia251@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Magnit251]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[praneetha.mullapudi@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[hi]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Don't sleep in class]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[C:\Users\prane\OneDrive\Pictures\Saved Pictures\ladies_maha_shiva.jpg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Tests.MailSentTestwithDp.mailSentTest(MailSentTestwithDp.java:122)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-19.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-19.png"height="100"width="100"/></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- mailSentTest -->
        <test-method status="FAIL" signature="mailSentTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:Tests.MailSentTestwithDp@e874448]" name="mailSentTest" duration-ms="40278" started-at="2021-03-21T20:59:20 IST" data-provider="providerMethod" finished-at="2021-03-21T21:00:01 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[magnitia251@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Magnit251]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[praneetha.mullapudi@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[helo]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[take care]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[C:\Users\prane\OneDrive\Pictures\Saved Pictures\lavender.jpg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Tests.MailSentTestwithDp.mailSentTest(MailSentTestwithDp.java:122)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-57.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-08-59-57.png"height="100"width="100"/></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- mailSentTest -->
        <test-method status="FAIL" signature="mailSentTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:Tests.MailSentTestwithDp@e874448]" name="mailSentTest" duration-ms="10522" started-at="2021-03-21T21:00:01 IST" data-provider="providerMethod" finished-at="2021-03-21T21:00:11 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[magnitia251@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Magnit251]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[praneetha.mullapudi@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[helo]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Happy Sivarathiri]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[C:\Users\prane\OneDrive\Pictures\Saved Pictures\lavender.jpg]]>
              </value>
            </param>
          </params>
          <exception class="io.github.bonigarcia.wdm.config.WebDriverManagerException">
            <message>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: msedgedriver.azureedge.net]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: msedgedriver.azureedge.net
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:776)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:573)
at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:792)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:773)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:573)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:766)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:573)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:287)
at UtilityClasses.TestUtility.OpenBrower(TestUtility.java:65)
at Tests.MailSentTestwithDp.mailSentTest(MailSentTestwithDp.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.net.UnknownHostException: msedgedriver.azureedge.net
at io.github.bonigarcia.wdm.WebDriverManager.getDriversFromXml(WebDriverManager.java:915)
at io.github.bonigarcia.wdm.managers.EdgeDriverManager.getDriverUrls(EdgeDriverManager.java:106)
at io.github.bonigarcia.wdm.WebDriverManager.createUrlHandler(WebDriverManager.java:797)
at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:649)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:567)
... 36 more
Caused by: java.net.UnknownHostException: msedgedriver.azureedge.net
at java.net.InetAddress.getAllByName0(Unknown Source)
at java.net.InetAddress.getAllByName(Unknown Source)
at java.net.InetAddress.getAllByName(Unknown Source)
at org.apache.hc.client5.http.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:43)
at org.apache.hc.client5.http.impl.io.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:118)
at org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:413)
at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:164)
at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:174)
at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:135)
at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:169)
at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:93)
at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:128)
at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:116)
at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:178)
at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:75)
at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:89)
at io.github.bonigarcia.wdm.online.HttpClient.execute(HttpClient.java:139)
at io.github.bonigarcia.wdm.WebDriverManager.getDriversFromXml(WebDriverManager.java:900)
... 40 more
]]>
            </full-stacktrace>
          </exception> <!-- io.github.bonigarcia.wdm.config.WebDriverManagerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mailSentTest -->
        <test-method status="FAIL" signature="mailSentTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:Tests.MailSentTestwithDp@e874448]" name="mailSentTest" duration-ms="34745" started-at="2021-03-21T21:00:11 IST" data-provider="providerMethod" finished-at="2021-03-21T21:00:46 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[opera]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[magnitia251@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Magnit251]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[praneetha.mullapudi@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[helo]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Flighting]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[C:\Users\prane\OneDrive\Pictures\Saved Pictures\lavender.jpg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Tests.MailSentTestwithDp.mailSentTest(MailSentTestwithDp.java:122)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-09-00-44.png"><img src="F:\batch 251\selenium_framworks\Frameworks\org.magnitia.5TDD\target\21-Mar-2021-09-00-44.png"height="100"width="100"/></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- mailSentTest -->
        <test-method status="PASS" signature="method4()[pri:0, instance:Tests.MailSentTestwithDp@e874448]" name="method4" is-config="true" duration-ms="2105" started-at="2021-03-21T21:00:46 IST" finished-at="2021-03-21T21:00:48 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method4 -->
      </class> <!-- Tests.MailSentTestwithDp -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
